mava/environment_loop.py

```diff
@@ -291,6 +292,26 @@ def step_executor() -> None:
                     results.update(eval_result)
                     self._logger.write(results)
 
+                    if self._executor.store.checkpoint_best_perf:
+                        # Best_performance_update
+                        for (
+                            metric,
+                            best_performance,
+                        ) in self._executor.store.metrics_checkpoint.items():
+                            assert (
+                                metric in results.keys()
+                            ), f"The metric chosen to checkpoint it best performance doesn't exist.\
+                                This experiment has only the following metrics {results.keys()}"
```

Comment:
Grammar: The metric, {metric}, chosen for checkpointing doesn't exist. This experiment has only the following metrics {results.keys()}

Reply:
Thank you! Will make the change!

Reply:
Done!